### ISSUE TITLE
OpenDistro 1.13.0 - GET /_cat/health endpoint getting 500 security exception

### ISSUE THREAD

**Issue by verbecee** [opendistro-for-elasticsearch/security-kibana-plugin#720]
Hi ODFE team. We are in the process of upgrading to 1.13.0, and in testing, we cannot run the GET /_cat/health command without getting the following response:

`{
  "error" : {
    "root_cause" : [
      {
        "type" : "security_exception",
        "reason" : "Unexpected exception cluster:monitor/health"
      }
    ],
    "type" : "security_exception",
    "reason" : "Unexpected exception cluster:monitor/health"
  },
  "status" : 500
}`

I get this 500 exception everywhere:
- running it from the command line, as the local admin
- any user with any level of permissions
- in the kibana console as an admin

This problem persists when running the single node independent elastic docker command in the [ODFE docs](https://opendistro.github.io/for-elasticsearch-docs/docs/install/docker/#run-the-image):
"docker run -d -p 9200:9200 -p 9600:9600 -e "discovery.type=single-node" --net=host amazon/opendistro-for-elasticsearch:1.13.0"
In other words, this isn't just an "our environment/customizations" issue.

However, I can run the following commands with no issue:
- GET /_cluster/health
- GET /_cat/nodes
- GET /_cat/shards
- POSTing docs work
- PUTting indexes also work

Was this endpoint deprecated? I searched the release notes and found nothing about that.

**Comment by trexx** [comment]
Likely to do with this bug https://github.com/opendistro-for-elasticsearch/security/issues/1042 which was fixed here https://github.com/opendistro-for-elasticsearch/security/pull/1048

You can build 1.13.1.0-rc1 or wait for it to be officially released.

**Comment by verbecee** [comment]
It does! Thank you so much, apologies for not seeing this earlier.
